pkg/apiserver/config.go

```diff
@@ -20,6 +21,11 @@ type Timeout struct {
 	ExcludedRoutes []string `mapstructure:"excluded_routes"`
 }
 
+type Logging struct {
+	// The list of routes that are excluded from the logging
+	ExcludedApiRoutes []string `mapstructure:"excluded_api_routes"`
```

Comment:
```suggestion
	ExcludedRoutes []string `mapstructure:"excluded_routes"`
```

We should be as consistent as possible. We used ExcludedRoutes for Timeout, let's use the same here.